src/Symfony/Component/Config/Definition/ArrayNode.php

```diff
@@ -48,7 +48,7 @@ public function setNormalizeKeys($normalizeKeys)
      *
      * @param mixed $value
      *
-     * @return array The value with normalized keys
+     * @return mixed The value with normalized keys
```

Comment:
maybe use `{@inheritdoc}` while overwriting the long/short desc

Reply:
I'm not sure we do this much, do we?

Reply:
see e.g. https://github.com/symfony/symfony/blob/6c9b87c5ea869dca35244a516b54e5863e3b5ce7/src/Symfony/Component/Validator/Mapping/Factory/LazyLoadingMetadataFactory.php#L64-L79

Reply:
updated, thanks